### PR TITLE
Fix wording and code example (client side application)

### PR DESCRIPTION
**Pull request by martinsam**
ping @frankrousseau

### CODE REVIEW

File: source/clientsideapp.md [hunk 1]
```diff
@@ -43,26 +43,33 @@ Here we are, doing the traditionnal "Hello World" app that can make you want to
   "displayName": "Your App's Name",
   "version": "1.0.0",
   "description": "Your app's description",
-  "main": "index.html"
+  "icon-path": "main_icon.png",
+  "author": "Your name",
```
Comment (jsilvestre) [inline source/clientsideapp.md]: Is author a mandatory field? Is it used somewhere by Cozy?

**Comment by jsilvestre** [comment]
Thank you for the fixes! I only have one comment, you may know better than me. Could you address it? Thanks!

File: source/clientsideapp.md (hunk 1 above)
Comment (martinsam) [inline source/clientsideapp.md]: I don't know.. so there is an example on https://github.com/cozy/cozysdk-client-tuto/blob/v1.0/package.json

Comment (martinsam) [inline source/clientsideapp.md]: In my opinion, It is important to have a consistency between documentation examples and github examples.

**Comment by martinsam** [comment]
I don't understand " Could you address it? "

**Comment by jsilvestre** [comment]
Est-ce que tu peux répondre à mon commentaire s'il te plait ;-)

**Comment by martinsam** [comment]
Ici : 
https://github.com/cozy-labs/cozy-dev-docs/pull/8#discussion_r53464977

**Comment by jsilvestre** [comment]
Thank you very much! We decided to remove the author field since it's not used at all, thus should not be documented. I opened a ticket in the right repository to mention it should be removed there too.

**Pull request merged**